ArgoProject/src/systems/CurrentGroundSystem.cpp

```diff
@@ -31,5 +32,18 @@ void app::sys::CurrentGroundSystem::update(app::time::seconds const & dt)
 				ground.currentGround.reset();
 			}
 		}
+		else 
+		{
+			if (m_registry.has<comp::Input>(entity))
```

Comment:
Instead of querying the registry for the `.has<comp::Input>()` function.
instead make a `.view<comp::Input>()` at the start of the update function and replace this line with
```suggestion
			if (inputView.contains(entity))
```